api_tests/e2e/rfc003/btc_eth/test.js

```diff
@@ -24,12 +24,14 @@ const logger = test_lib.logger();
 describe("RFC003: Bitcoin for Ether", () => {
     before(async function() {
         this.timeout(5000);
+        await test_lib.btc_activate_segwit();
+        test_lib.btc_enable_generate_every(1000);
         await bob.wallet.fund_eth(bob_initial_eth);
         await alice.wallet.fund_eth(alice_initial_eth);
         await alice.wallet.fund_btc(10);
-        await test_lib.import_address(bob_final_address); // Watch only import
-        await test_lib.import_address(alice.wallet.btc_identity().address); // Watch only import
-        await test_lib.import_address(bob.wallet.btc_identity().address); // Watch only import
+        await test_lib.btc_import_address(bob_final_address); // Watch only import
```

Comment:
ℹ️ I made a ticket about removing this: https://github.com/comit-network/comit-rs/issues/566